src/components/Textfield/textfield.scss

```diff
@@ -198,3 +198,21 @@
     }
   }
 }
+
+.textbox {
+  &.has-error {
+    border: 1px solid var(--danger-color) !important;
+    .contacto-select:hover:not(.ant-select-disabled),
+    .contacto-select[class*='focused']:not(.ant-select-disabled) {
+      border: none;
+    }
+    .contacto-select {
+      border: none;
+      box-shadow: none;
+    }
```

Comment:
**Suggestion:** Avoid using !important in CSS. Try to increase specificity of the selector instead.
```suggestion
&.has-error {
  border: 1px solid var(--danger-color);
  .contacto-select:hover:not(.ant-select-disabled),
  .contacto-select[class*='focused']:not(.ant-select-disabled) {
    border: none;
  }
  .contacto-select {
    border: none;
    box-shadow: none;
  }
}
```